pkg/appmgmt/general/podevent_handler.go

```diff
@@ -127,6 +130,15 @@ func (p *PodEventHandler) addPod(pod *v1.Pod, eventSource EventSource) interface
 				Metadata: appMeta,
 			})
 		} else {
+			if conf.GetSchedulerConf().GetSingleUserPerApplication() && app.GetUser() != appMeta.User {
```

Comment:
This is not good enough as it will break the autogen IDs. If we generate one ID per namespace the system comes to a grinding halt. That behaviour is managed by the setting in `admissionController.filtering.generateUniqueAppId` if that is false (default) all pods that miss an app ID will have it set to: `yunikorn-NAMESPACE-autogen` that could be for multiple users.

Reply:
We would skip the check for auto gen app id's